WcaOnRails/app/assets/stylesheets/users.scss

```diff
@@ -19,3 +19,16 @@ input[type="radio"].center-vertically {
   vertical-align: middle;
   height: 100%;
 }
+
+.input-group.date.datetimepicker {
+  width: 100%;
+}
+
+.form-group {
```

Comment:
We moved away from the button because we both thought that was the only way to get the calendar to pop up when you focus an `<input>`. I'm pretty sure one of those options I linked you to will allow us to do both, however.